infrastructure/containers/postgres/automatic-backup/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN python3 -m pip install boto3 xonsh
 WORKDIR /app
 
 COPY backup_or_restore.py ./backup_or_restore.py
+COPY scripts/startup-script.sh ./scripts/startup-script.sh
+COPY scripts/wait-for-postgres-init.sh ./scripts/wait-for-postgres-init.sh
```

Comment:
Awesome! I love that you both copied these scripts into the Dockerfile _and_ mounted them in the `docker-compose.yml`.